docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>Worth running black on this. It should be done in the regular pre-commit I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yeah, this runs at each pre-commit check